build-system/tasks/e2e/package.json

```diff
@@ -6,7 +6,7 @@
   "devDependencies": {
     "@babel/register": "7.5.5",
     "babel-regenerator-runtime": "6.5.0",
-    "chromedriver": "74.0.0",
+    "chromedriver": "76.0.0",
```

Comment:
This needs a corresponding change to `build-system/tasks/e2e/yarn.lock`, doesn't it?

Reply:
i did run yarn, but nothing showed up. Assumed that it might be in gitignore

Reply:
Aha. It's possible you did so from the protect root dir instead of the directory containing the lockfile?

Reply:
I can add it with this https://github.com/ampproject/amphtml/pull/23626

Reply:
that..... might be it